packages/core/src/telemetry/service.test.ts

```diff
@@ -8,17 +8,14 @@ import { TelemetryService } from "@/telemetry/service";
 
 const conf = new Conf({ projectName: "ponder" });
 
-let latestFetchBody: any = undefined;
-const fetchSpy = vi.fn().mockImplementation((_url, args) => {
-  latestFetchBody = JSON.parse(args.body);
-});
+const fetchSpy = vi.fn().mockImplementation(() => vi.fn());
```

Comment:
there is no need to store the fetch body in a separate variable if we can access the body with `fetchSpy.mock.calls`

Also, this doesn't work if we reset the mock in the `beforeEeach` hook. We do this to reset the function calls count between tests.